grpc/errors.go

```diff
@@ -38,13 +39,21 @@ func wrapError(ctx context.Context, err error) error {
 			jsonSubErrs, err := json.Marshal(berr.SubErrors)
 			if err != nil {
 				return berrors.InternalServerError(
+					0,
 					"error marshaling json SubErrors, orig error %q",
 					err)
 			}
 			pairs = append(pairs, "suberrors")
 			pairs = append(pairs, string(jsonSubErrs))
 		}
 
+		// If there is a RetryAfter value then extend the metadata pairs to
+		// include the value.
+		if berr.RetryAfter > 0 {
+			pairs = append(pairs, "retryafter")
+			pairs = append(pairs, berr.RetryAfter.String())
```

Comment:
Technically this could be
```suggestion
			pairs = append(pairs, "retryafter", berr.RetryAfter.String())
```
because `append(...)` takes arbitrarily many items to append. I think this makes the map-like nature of `pairs` a little clearer, but it's different from how the suberrors are added, so totally up to you.

Reply:
I've gone ahead and done the same with 'suberrors'.